tools/obscuroscan/obscuroscan_test.go

```diff
@@ -39,6 +41,7 @@ func TestThrowsIfEncryptedRollupIsInvalid(t *testing.T) {
 
 // Generates an encrypted transaction blob in Base64 encoding.
 func generateEncryptedTxBlob(txs []*common.L2Tx) []byte {
-	txBlob := crypto.NewTransactionBlobCryptoImpl().Encrypt(txs)
+	rollup := core.Rollup{Header: &common.Header{}, Transactions: txs}
```

Comment:
These changes allow us to get around the removal of the `Encrypt` method from the public API.